### ISSUE TITLE
How to add url/website to whitelist and all its sub-links and continue to block trackers, ads, and other filtering

### ISSUE THREAD

**Issue by YesTrustMe** [AdguardTeam/AdGuardMV3#65]
### Please answer the following questions for yourself before submitting a question.

- [X] I am running the latest version
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Ask a question

On one site, eg. https://acme.com/, I am now blocked via extension from accessing the site.

I want to add acme.com and acme.com/list1 and acme.com/list2 etc but I also want to [continue] to stop the background "noise" including ads, trackers, continue to block cosmetic filtering (just like before it was blocked by Adguard). 

Basically, a new Adguard update added the site to its blacklist and I played with the existing settings without any success.

**Comment by alexx7311** [comment]
@YesTrustMe Hello! If a website has been incorrectly blocked by our extension or it is not working properly, you can report it via 'Report an issue' button in the popup and our Filters team will look at it.
<details>
  <summary>Screenshot</summary>

  ![Screenshot](https://github.com/AdguardTeam/AdGuardMV3/assets/114913601/1717e218-cd40-4c1d-9f28-55f7d4a613a2)

</details>

If you want to try creating exceptions for certain requests yourself, you can use our knowledge base article at https://adguard.info/kb/general/ad-filtering/create-own-filters/
